src/Test.UnitTests.Sarif/TestData/PrereleaseCompatibilityTransformer/ExpectedOutputs/DateTimeInPropertyBag.sarif

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/jsons/sarif-2.1.0-rtm.4.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner"
+        }
+      },
+      "invocations": [
+        {
+          "startTimeUtc": "2019-07-09T09:56:46.144Z",
+          "executionSuccessful": true
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "message": {
+            "text": "Message."
+          },
+          "properties": {
+            "alertDiscoveryDate": "2019-06-22T19:06:31.209Z"
```

Comment:
>"2019-06-22T19:06:31.20 [](start = 34, length = 23)

This bug fix and problem is too specifics and small to warrant its own file unless that file is defined/design to eventually absorb some broader spectrum of testing. Can you put some thought into it?

Reply:
Good idea. I combined the two single-issue test files DateTimeInPropertyBag.sarif for #1577, and Rtm1SchemaLocation.sarif for #1584, into a single file RegressionTests.sarif, and updated the test methods accordingly. See the diff between iterations 8 and 9.

---
In reply to: [316737193](https://github.com/microsoft/sarif-sdk/pull/1651#discussion_r316737193) [](ancestors = 316737193)